client/cypress/integration/query/parameter_spec.js

```diff
@@ -265,14 +277,11 @@ describe('Parameter', () => {
       cy.getByTestId('ParameterApplyButton')
         .click();
 
-      cy.get('@Input').then(($input) => {
-        const now = Cypress.moment($input.val(), 'DD/MM/YY HH:mm');
-        cy.getByTestId('DynamicTable')
-          .should('contain', now.format('YYYY-MM-15 HH:mm'));
-      });
+      cy.getByTestId('DynamicTable')
+        .should('contain', Cypress.moment(this.now).format('YYYY-MM-15 HH:mm'));
```

Comment:
@ranbena and @rauchy I updated a bit the Cypress date assertion, `Cypress.moment()` is not affected by `cy.clock()`, so I put the time selected as an alias and took it from `this.now`.